nixos/modules/services/misc/snapper.nix

```diff
@@ -78,6 +78,54 @@ let
         Defines whether hourly snapshots should be created.
       '';
     };
+
+    TIMELINE_LIMIT_HOURLY = mkOption {
```

Comment:
The format for NixOS modules is: `timelineLimitHourly`

Reply:
For what I understand, options are copied "as is" in the configuration file.

```
        // (mapAttrs'
          (name: subvolume: nameValuePair "snapper/configs/${name}" ({
            text = lib.generators.toKeyValue { inherit mkKeyValue; } (filterAttrs (k: v: v != defaultOf k) subvolume);
          }))
          cfg.configs)
```

End I'm not sur I want to change this magic line.

Reply:
Right. Nevermind.